packages/merge-on-green/src/merge-logic.ts

```diff
@@ -240,9 +244,10 @@ mergeOnGreen.iterateGetStatusi = async function iterateGetStatusi(
   let results: CheckStatus[] = [];
   for (let i = 0; i < 10; i++) {
     const temp = await mergeOnGreen.getStatusi(owner, repo, github, headSha, i);
-    if (temp.length !== 0) {
-      results = results.concat(temp);
+    if (temp.length === 0) {
+      return results;
```

Comment:
Not sure I understand this line -- return early if there is no more `getStatusi`?

Reply:
Yep, exactly. Basically, check to make sure that that page returns 0 results, and then return whichever results you've already accumulated in "results"